<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="q_btn_mastermode">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>250</width>
      <height>400</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Master Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_btn_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>250</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/btn/111.png);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_btn_status">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>250</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_btn_stop">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>450</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
